src/coreclr/jit/compiler.h

```diff
@@ -5358,6 +5358,22 @@ class Compiler
 
     PhaseStatus fgCloneFinally();
 
+    struct CloneTryInfo
+    {
+        CloneTryInfo(BitVecTraits& traits, BitVec& visited) : m_traits(traits), m_visited(visited) {}
+        BitVecTraits m_traits;
+        BitVec& m_visited;
+        BlockToBlockMap* m_map = nullptr;
+        jitstd::vector<BasicBlock*>* m_blocksToClone = nullptr;
+        bool m_addEdges = false;
+        weight_t m_profileScale = 0.0;
+        unsigned m_ehRegionShift = 0;
```

Comment:
The `m_` prefix looks a bit odd to me for public fields meant to be accessed externally.

Reply:
Ok, changed to be initial caps.